### PR TITLE
Add early bird price

### LINKED ISSUE #1011: ✨ Add early bird price field to component `eventBooking`
**Is your feature request related to a problem? Please describe.**

On some training page, the price of events have early bird price.

Need to add field on Event Booking component.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by johnxu16**
Fixes #1011 

Affected routes:

- /training/internship-fullstack

Add done video, screenshots

image
**Figure: early bird price field on tinacms*

### CODE REVIEW

File: components/training/eventBooking.tsx [hunk 1]
```diff
@@ -165,8 +166,23 @@ const EventHeader = ({ duration, price, schema }) => {
...
             {EventModel.PRICE}
           </div>
-          {EventModel.CURRENCY}
-          <span>{price}</span> {EventModel.INCLUDE_GST}
+          <span
+            className={classNames({
+              "text-gray-450 line-through":
+                earlyBirdPrice !== undefined &&
+                earlyBirdPrice !== null &&
+                earlyBirdPrice !== -1,
```
Comment (amankumarrr) [inline components/training/eventBooking.tsx]: Shouldn't be a function that returns this as you're using it somewhere else as well?

File: components/training/eventBooking.tsx [hunk 2]
```diff
@@ -177,6 +193,7 @@ export const eventBookingBlock = {
   eventBooking: "EventBooking",
   duration: "duration",
   price: "price",
+  earlyBirdPrice: "earlyBirdPrice",
```
Comment (amankumarrr) [inline components/training/eventBooking.tsx]: Shouldn't it be discounted price? "Early bird" text should be dynamically added

**Review by amankumarrr — CHANGES_REQUESTED** [review]

File: components/training/eventBooking.tsx (hunk 1 above)
Comment (johnxu16) [reply components/training/eventBooking.tsx]: Done - create function isEmpty

File: components/training/eventBooking.tsx (hunk 2 above)
Comment (johnxu16) [reply components/training/eventBooking.tsx]: For better customization, I changed earlyBirdPrice to discountPrice and changed its type into string as well

**Review by amankumarrr — APPROVED** [review]
LGTM 🌟

**Pull request merged**